jupyter-notebooks/orders_basemaps/SDK_order_basemaps.ipynb

```diff
@@ -0,0 +1,403 @@
+    "}\n",
+    "\n",
+    "basemap_service_response = requests.get(url=BASEMAP_API_URL, params=basemap_params, auth=auth, headers=headers)\n",
+    "\n",
+    "basemaps = basemap_service_response.raise_for_status()\n",
+    "if basemap_service_response.status_code != 204:\n",
+    "    basemaps = json.loads(basemap_service_response.text)\n",
+    "print(basemaps)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mosaic_ID = basemaps['mosaics'][0]['id']\n",
+    "mosaic_box = basemaps['mosaics'][0]['bbox']\n",
+    "mosaic_box_str = ','.join(map(str, mosaic_box))\n",
+    "\n",
+    "quad_params = {\n",
+    "    'mosaic_id': mosaic_ID,\n",
+    "    'bbox': mosaic_box_str,\n",
+    "}\n",
+    "\n",
+    "quads_url = \"{}/{}/quads\".format(BASEMAP_API_URL, mosaic_ID)\n",
+    "quadServiceResponse = requests.get(\n",
+    "    url=quads_url, params=quad_params, auth=auth, headers=headers)\n",
+    "quads = quadServiceResponse.json()\n",
+    "items = quads['items']\n",
+    "\n",
+    "quadIds = []\n",
+    "for i in items:\n",
+    "    quadId = i['id']\n",
+    "    quadIds.append(quadId)\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Create an order packet\n",
+    "---\n",
+    "Package up the details of your order in a JSON object and make a POST request, passing in the Orders URL, your JSON, your API key, and the content-type header. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "order_params = {\n",
+    "    \"name\": \"basemap order with quad_ids\",\n",
+    "    \"source_type\": \"basemaps\",\n",
+    "    \"products\": [\n",
+    "        {\n",
+    "            \"mosaic_name\": \"point_reyes_rolling_normalized_2020-04-30_mosaic\",\n",
+    "            \"quad_ids\": quadIds\n",
+    "        }\n",
+    "    ],\n",
+    "    \"tools\": [\n",
+    "        {\n",
+    "            \"reproject\": {\n",
+    "                \"projection\": \"EPSG:4326\",\n",
+    "                \"resolution\": 3.0 / 111000.0,\n",
+    "                \"kernel\": \"cubic\"\n",
+    "            }\n",
+    "        },\n",
+    "        {\n",
+    "            \"bandmath\": {\n",
+    "                \"b1\": \"b1\",\n",
+    "                \"b2\": \"b2\",\n",
+    "                \"b3\": \"b3\",\n",
+    "                \"b4\": \"arctan(b1)\",\n",
+    "                \"b5\": \"(b4-b3)/(b4+b3)\",\n",
+    "                \"pixel_type\": \"32R\"\n",
+    "            }\n",
+    "        }\n",
+    "    ],\n",
+    "    \"delivery\": {\n",
+    "        \"google_cloud_storage\": {\n",
+    "            \"bucket\": \"devrel-notebooks\",\n",
+    "            \"credentials\": GCP_CREDENTIALS,\n",
+    "            \"path_prefix\": \"basemaps-to-cloud/\",\n",
+    "        }\n",
+    "    }\n",
```

Comment:
How will people know where this structure comes from? Make sure you're linking to the documentation here so folks don't get lost in all the brackets :D